rebasehelper/spec_hooks/typo_fix.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import re
+
+from rebasehelper.specfile import BaseSpecHook
+
+
+class TypoFixHook(BaseSpecHook):
+    """Sample spec hook that fixes typos in spec file"""
+
+    NAME = 'Typo fix'
+    REPLACEMENTS = [
+        ('chnagelog', 'changelog'),
+        ('indentional', 'intentional'),
+    ]
```

Comment:
This was actually meant to be just a trivial sample spec hook. But perhaps you are right, and it could actually be useful. But in that case it should probably take advantage of some dictionary of typos, like [this one from Wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Lists_of_common_misspellings), and also some logic to prevent false positives should be implemented.